src/main/scala/org/mbari/vars/annotation/api/v1/ImageV1Api.scala

```diff
@@ -130,10 +130,11 @@ class ImageV1Api(controller: ImageController)(implicit val executor: ExecutionCo
       )
     }
 
-    val format      = params.get("format")
-    val width       = params.getAs[Int]("width_pixels")
-    val height      = params.getAs[Int]("height_pixels")
-    val description = params.get("description")
+    val format             = params.get("format")
+    val width              = params.getAs[Int]("width_pixels")
+    val height             = params.getAs[Int]("height_pixels")
+    val description        = params.get("description")
+    val imageReferenceUUID = params.getAs[UUID]("image_reference_uuid")
```

Comment:
@kevinsbarnard Just some notes to explain the code. Here, imageReferenceUUID is an [Option](https://www.scala-lang.org/api/current/scala/Option.html). Analagous to [Optional](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Optional.html) in Java and Optional in python (which is actually to Union[X, None].). It's one way to deal with possible null values. This line could be also be written with the type signature as `val imageReferenceUUID: Option[UUID] = params.getAs[UUID]("image_reference_uuid")`